src/cloud/trouble/trouble_comp-deploy-fail.md

```diff
@@ -1,56 +1,51 @@
 ---
 group: cloud-guide
-subgroup: 170_trouble
 title: Component deployment failure
-menu_title: Component deployment failure
-menu_order: 15
-menu_node:
 functional_areas:
   - Cloud
   - Deploy
 ---
 
-This topic discusses how to recover from a failed component deployment. Typical examples are components that have dependencies that are not met by your environment; for example, incompatible [PHP](https://glossary.magento.com/php) versions.
+This topic discusses how to recover from a failed component deployment. Typical examples include components that have dependencies that are not met by your environment, such as incompatible PHP versions.
 
 You can recover from a failed deployment in any of the following ways:
 
-*  [Restore a snapshot]({{ site.baseurl }}/cloud/project/project-webint-snap.html) if you have one
-*  Remove the component from your environment's `composer.json` and redeploy the environment
+-  [Restore a snapshot]({{ site.baseurl }}/cloud/project/project-webint-snap.html)
+-  Remove the component from the `composer.json` and redeploy the environment
 
-## Remove the component from `composer.json` and redeploy
+## Clean, remove, and redeploy
 
-This section discusses how to remove the component from the root `composer.json` in your environment and redeploy the environment:
+To clean up from the previous deployment, you must log in to the remote environment and manually clear the contents of the Magento `var` directory. And then you must remove the component from the `composer.json` file in your local environment and redeploy.
 
-## Get started
+{:.procedure}
+To clean the `var` directories:
 
-{% include cloud/cli-get-started.md %}
+1. On your local workstation, change to the Cloud project root directory.
 
-## Find a component's Composer name
-
-{% include cloud/composer-name.md %}
-
-## Clear `var` directories
-
-To clean up from the previous deployment, you must SSH to the environment and manually clear the contents of the Magento `var` directory.
-
-1. Enter the following command to SSH to the current environment:
+1. Use a SSH to log in to the remote environment.
 
    ```bash
    magento-cloud environment:ssh
    ```
 
-1. Clear the `var` directory:
+1. Clear the `var` directories.
 
-   ```bash
+   ```shell
    rm -rf var/*
    ```
 
-## Remove the component
-
+{:.procedure}
 To remove the component:
 
-1. Change to your environment's root directory if you haven't already done so.
-1. Enter the following command:
+1. On your local workstation, change to the Cloud project root directory.
+
+1. Clear the cache.
+
+   ```bash
+   composer clearcache
```

Comment:
`clear-cache` is base variant

Reply:
okay, what about `cc`? Just kidding. I'll change it.